### PR TITLE
 KAFKA-4627: Fix timing issue in consumer close tests

### PR DESCRIPTION
**Pull request by rajinisivaram**

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@hachikuji Can you take a look please? Thank you.

File: clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java [hunk 1]
```diff
@@ -1560,7 +1560,8 @@ private void close(long timeoutMs, boolean swallowException) {
         AtomicReference<Throwable> firstException = new AtomicReference<>();
         this.closed = true;
         try {
-            coordinator.close(Math.min(timeoutMs, requestTimeoutMs));
+            if (coordinator != null)
```
Comment (hachikuji) [inline clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java:1563]: Good catch.

**Review by apurvam — APPROVED** [review]
LGTM!

**Comment by hachikuji** [comment]
LGTM

**Pull request closed without merging**